src/DotNetWorker.Core/Logging/NullLogWriter.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Functions.Worker.Logging
+{
+    /// <summary>
+    /// Minimalistic LogWriter that does nothing.
```

Comment:
Could you elaborate on this LogWriter? Trying to understand it. Is it for when a user doesn't create/define other specific loggers (UserLogWriter, SystemLogWriter) and this is the default put in their place? 

This summary caught me off guard 🤣

Reply:
All of our "logging through the host" goes through Grpc, which isn't part of the core packages. So we need some default way to basically "do nothing" without breaking dependency injection if for some reason Grpc isn't wired up. In reality this will be very rare.